libs/platform/table/components/table-view-settings-dialog/grouping/grouping.component.html

```diff
@@ -17,9 +17,15 @@ <h4 fd-title [headerSize]="4">{{ 'platformTable.groupDialogHeader' | fdTranslate
     <fd-dialog-body disablePaddings>
         <ul fd-list [selection]="true">
             <li fd-list-group-header>
-                <span fd-list-title>{{ 'platformTable.groupDialogGroupOrderHeader' | fdTranslate }}</span>
+                <span fd-list-title id="group-dialog-group-order-header">{{
```

Comment:
in my opinion, this id should be auto generated and then assigned as aria-describedby to the list items (also automatically).